.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Reload Data (Staging Environment)
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "**.csv"
+  workflow_dispatch:
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Upload file to bucket
+        uses: Hyperobjekt/s3-upload-github-action@master
+        with:
+          args: --acl public-read
+        env:
+          FILE: "./data/tulsa_eviction_cases.csv"
+          AWS_REGION: "us-east-1"
+          S3_BUCKET: "tulsastack-stagingdatastore42145388-1bvtz4nrc4k1i"
+          S3_KEY: "source.csv"
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Make sure these two secrets are set up for the repository exactly as they are here